src/goto-programs/goto_program.h

```diff
@@ -529,6 +529,23 @@ class goto_programt
     return t;
   }
 
+  optionalt<const_targett> get_target(const unsigned location_number) const
```

Comment:
We have elsewhere erased the `const` qualifier from POD parameters, so it's probably better to drop it here.

Reply:
IMHO this is good to keep on definitions, as it still meaningful "I'm not going to modify this value in this body". I'd only remove it from declarations, where it is meaningless.